cypress/integration/buy-shirt.spec.ts

```diff
@@ -19,7 +25,7 @@ describe('Buy a t-shirt', () => {
     cy.get("a.bankwire").click();
     cy.get(".cart_navigation span").click();
 
-    cy.get('#center_column > div > p > strong')
-      .should('have.text', 'Your order on My Store is complete.')
+    cy.get("#center_column > div > p > strong")
+      .should("have.text", "Your order on My Store is complete.")
```

Comment:
Use the page objects that you created to replace these lines by calls to those objects